src/common/utils.ts

```diff
@@ -50,5 +52,5 @@ export async function promisify(funThis, functionName, ...args): Promise<any> {
 };
 
 export function obfuscate(text: string): string {
-  return text ? `${[...Array(text.length - 2)].map(()=>'x').join('')}${text.substr(text.length -2)}` : text;
-}
\ No newline at end of file
+  return text ? `${[...Array(text.length - 2)].map(() => 'x').join('')}${text.substr(text.length - 2)}` : text;
```

Comment:
```suggestion
  return text
    ? `${[...Array(text.length - 2)].map(() => 'x').join('')}${text.substr(text.length - 2)}`
    : text;
```

Should we add a small description to this function? It' also breaks the grid rulers of max characters 100.